<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblDeviceSelector">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbDeviceSelector"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QToolButton" name="btnTruth">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-top:0px;
padding-bottom:0px;
margin-top:0px;
margin-bottom:0px;
spacing:0px;</string>
        </property>
        <property name="text">
         <string>Truth Table</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/sideIcon/applications-development.png</normaloff>:/sideIcon/applications-development.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAlgebra">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-top:0px;
padding-bottom:0px;
margin-top:0px;
margin-bottom:0px;
spacing:0px;</string>
        </property>
        <property name="text">
         <string>Algebra</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/sideIcon/applications-development.png</normaloff>:/sideIcon/applications-development.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAnalyse">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-top:0px;
padding-bottom:0px;
margin-top:0px;
margin-bottom:0px;
spacing:0px;</string>
        </property>
        <property name="text">
         <string>Analyse</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/sideIcon/applications-development.png</normaloff>:/sideIcon/applications-development.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCircuit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>padding-top:0px;
padding-bottom:0px;
margin-top:0px;
margin-bottom:0px;
spacing:0px;</string>
        </property>
        <property name="text">
         <string>Circuit</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/sideIcon/applications-development.png</normaloff>:/sideIcon/applications-development.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnVHDL">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>padding-top:0px;
padding-bottom:0px;
margin-top:0px;
margin-bottom:0px;
spacing:0px;</string>
        </property>
        <property name="text">
         <string>VHDL</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/sideIcon/applications-development.png</normaloff>:/sideIcon/applications-development.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>91</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frmTruth">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QSpinBox" name="spnInputNum">
           <property name="suffix">
            <string> input</string>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnOutputNum">
           <property name="suffix">
            <string> output</string>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tblTruthTest">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>40</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frmCircuit">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sceneRect">
          <rectf>
           <x>0.000000000000000</x>
           <y>0.000000000000000</y>
           <width>1000.000000000000000</width>
           <height>1000.000000000000000</height>
          </rectf>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::RubberBandDrag</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCircuitSettings">
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frmAdvancedCircuitSettings">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="lblAndIn">
            <property name="text">
             <string>Maximum &quot;and&quot; input</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spnAnd"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblOrIn">
            <property name="text">
             <string>Maximum &quot;or&quot; input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spnOr"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblXorIn">
            <property name="text">
             <string>Maximum &quot;xor&quot; input</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spnXor"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="ckbPropDelay">
            <property name="toolTip">
             <string>Usually create bigger and slower circuit, but necessary in some situations. If you don't know what this is about, don't check this box.</string>
            </property>
            <property name="whatsThis">
             <string>Usually create bigger and slower circuit, but necessary in some situations. If you don't know what this is about, don't check this box.</string>
            </property>
            <property name="text">
             <string>Take propagation delay into account</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frmAlgebra">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTextEdit" name="txtOutTest"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QFrame" name="frmAnalyse">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblInputNum">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbbInputVal">
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveImage">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnZoomOut">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldZoom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnZoomIn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graAnalyserData">
         <property name="sceneRect">
          <rectf>
           <x>0.000000000000000</x>
           <y>0.000000000000000</y>
           <width>3000.000000000000000</width>
           <height>1100.000000000000000</height>
          </rectf>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QFrame" name="frmIOMode">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblIO0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbbIO0">
         <item>
          <property name="text">
           <string>Disable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Input</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Output</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblIO1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbbIO1">
         <item>
          <property name="text">
           <string>Disable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Input</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Output</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QFrame" name="frmVHDL">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGraphicsView" name="graVHDL"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCircuitSettings</sender>
   <signal>clicked(bool)</signal>
   <receiver>frmAdvancedCircuitSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
